### PR TITLE
Update migrating.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Minor formatting adjustments.

### CODE REVIEW

File: source/administration/migrating.rst [hunk 1]
```diff
@@ -89,13 +89,13 @@ If your data in the bespoke messenger is vital, consider:
 
 1. `Mattermost Bulk Load tool <https://docs.mattermost.com/deployment/bulk-loading.html>`__ - Use the Mattermost Bulk Load tool to ETL from your bespoke system to Mattermost.
 2. `Mattermost ETL framework from BrightScout <https://github.com/Brightscout/mattermost-etl>`__- Consider the Mattermost ETL framework from BrightScout to custom-configure an adapter to plug in to the Bulk Load tool mentioned above.
```
Comment (amyblais) [inline source/administration/migrating.rst:91]: Why are the first to bullet points not formatted the same way as the below ones?

File: source/administration/migrating.rst [hunk 2]
```diff
@@ -89,13 +89,13 @@ If your data in the bespoke messenger is vital, consider:
 
 1. `Mattermost Bulk Load tool <https://docs.mattermost.com/deployment/bulk-loading.html>`__ - Use the Mattermost Bulk Load tool to ETL from your bespoke system to Mattermost.
```
Comment (justinegeffen) [inline source/administration/migrating.rst:90]:
```suggestion
1. `Mattermost Bulk Load tool <https://docs.mattermost.com/deployment/bulk-loading.html>`__: Use the Mattermost Bulk Load tool to ETL from your bespoke system to Mattermost.
```

File: source/administration/migrating.rst (hunk 1 above)
Comment (justinegeffen) [inline source/administration/migrating.rst:91]:
```suggestion
2. `Mattermost ETL framework from BrightScout <https://github.com/Brightscout/mattermost-etl>`__: Consider the Mattermost ETL framework from BrightScout to custom-configure an adapter to plug in to the Bulk Load tool mentioned above.
```

Comment (justinegeffen) [reply source/administration/migrating.rst:91]: Good question - I didn't actually notice that! I can't seem to add bolding without messing up the link formatting, but I have changed the dash to colons.

**Review by amyblais — APPROVED** [review]

**Pull request merged**